site/src/components/card/Card.module.css

```diff
@@ -19,13 +24,27 @@
   flex: 1;
 }
 
+.card:hover {
+  box-shadow: var(--uitk-overlayable-shadow-borderRegion);
```

Comment:
a design decision, but I feel --uitk-overlayable-shadow-hover would be more prominent here

Reply:
we've tried that one and decided this one is a bit more subtle but I'll leave it to @shey-v :)